kafka/0.5/Dockerfile

```diff
@@ -40,6 +40,7 @@ RUN echo "$MD5HASH  /tmp/kafka.tgz" | md5sum -c - &&\
     rm -f /tmp/kafka.tgz
 
 COPY ./log4j.properties $KAFKA_HOME/config/log4j.properties
+RUN mkdir $KAFKA_HOME/config.template && mv $KAFKA_HOME/config/* $KAFKA_HOME/config.template
```

Comment:
Can you explain the name `config.template`? Seems like it's more of a copy than templates.